<ui version="4.0" >
 <class>Cnr2Dialog</class>
 <widget class="QDialog" name="Cnr2Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Chroma Noise Reducer 2</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Scene Change</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>0%</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Scene Change Threshold</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>100%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="sceneChangeSlider" >
            <property name="maximum" >
             <number>10000</number>
            </property>
            <property name="singleStep" >
             <number>100</number>
            </property>
            <property name="pageStep" >
             <number>1000</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition" >
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="sceneChangeSpinBox" >
          <property name="maximum" >
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="useChromaCheckBox" >
        <property name="text" >
         <string>Use chroma to also detect scene change</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="2" column="3" >
         <widget class="QComboBox" name="modeUComboBox" >
          <item>
           <property name="text" >
            <string>Wide</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Narrow</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QSpinBox" name="maxVSpinBox" >
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Luma Y:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Chroma V:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3" >
         <widget class="QComboBox" name="modeYComboBox" >
          <item>
           <property name="text" >
            <string>Wide</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Narrow</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2" >
         <widget class="QSpinBox" name="maxUSpinBox" >
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Maximum</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Chroma U:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QComboBox" name="modeVComboBox" >
          <item>
           <property name="text" >
            <string>Wide</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Narrow</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QSpinBox" name="maxYSpinBox" >
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QSpinBox" name="sensibilityVSpinBox" >
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="sensibilityYSpinBox" >
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Sensibility</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QSpinBox" name="sensibilityUSpinBox" >
          <property name="maximum" >
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sceneChangeSlider</tabstop>
  <tabstop>sceneChangeSpinBox</tabstop>
  <tabstop>useChromaCheckBox</tabstop>
  <tabstop>sensibilityYSpinBox</tabstop>
  <tabstop>maxYSpinBox</tabstop>
  <tabstop>modeYComboBox</tabstop>
  <tabstop>sensibilityUSpinBox</tabstop>
  <tabstop>maxUSpinBox</tabstop>
  <tabstop>modeUComboBox</tabstop>
  <tabstop>sensibilityVSpinBox</tabstop>
  <tabstop>maxVSpinBox</tabstop>
  <tabstop>modeVComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Cnr2Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Cnr2Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
